src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2835,6 +2838,11 @@ void processContextElement
       responseP->contextElementResponseVector.push_back(cerP);
     }
   }
+  else if (apiVersion == "v2" && action == "APPEND" && !isUpdate)
```

Comment:
The broker supports two similar requests:

    POST /v2/entities              (service routine postEntities)
    POST /v2/entities/{EntityId}   (service routine postEntity)

The first one is meant for creating an entity and if it already exists, it should an error (the bug addressed in #1158).
In the first request, the field "id" must be present.

The second one is more about creating attributes, not entities.
No field "id" is allowed. If present, error.
Also, if no entity with that ID exists, you should get a 404 Not Found.

It (the second request) has a URI parameter, **op**, that can take one single value (or empty): "append".
If "op==append", then APPEND_STRICT is used, meaning that trying to append an attribute that already exists gives an error.